lib/executor/engine/variableInstr.cpp

```diff
@@ -38,7 +38,9 @@ Expect<void> Executor::runGlobalGetOp(Runtime::StoreManager &StoreMgr,
 Expect<void> Executor::runGlobalSetOp(Runtime::StoreManager &StoreMgr,
                                       const uint32_t Idx) {
   auto *GlobInst = getGlobInstByIdx(StoreMgr, Idx);
-  GlobInst->getValue() = StackMgr.pop();
+  if (GlobInst) {
```

Comment:
Please use `assuming` to check the `nullptr`.

Ref: https://github.com/WasmEdge/WasmEdge/blob/bee4ad9831dec02d3224b9005d69cf30e3e973f6/lib/executor/instantiate/data.cpp#L38

Reply:
okay sure, thanks for the reference.